### PR TITLE
DAOS-5456 control: Add support for space_rb/self_heal props

### PR DESCRIPTION
**Pull request by mjmac**
Adds support for more pool properties. Support for the label
property will come later.

Also cleans up some debug logging in the gRPC pool handlers.

### CODE REVIEW

**Comment by ravalsam** [comment]
Tried to test the basic scenario of disabling the rebuild for pool and that is working fine.

**Review by ravalsam — APPROVED** [review]
LGTM

File: src/control/server/mgmt_pool_test.go [hunk 1]
```diff
@@ -1078,6 +1078,135 @@ func TestServer_MgmtSvc_PoolSetProp(t *testing.T) {
 				},
 			},
 		},
+		// label not supported yet (needs a MS map for label -> UUID)
+		/*"label": {
```
Comment (kjacque) [inline src/control/server/mgmt_pool_test.go:1082]: Same comment on the commented-out code

**Review by kjacque — COMMENTED** [review]
Overall looks good.

File: src/control/server/mgmt_pool.go [hunk 2]
```diff
@@ -291,23 +291,51 @@ func resolvePoolPropVal(req *mgmtpb.PoolSetPropReq) (*mgmtpb.PoolSetPropReq, err
 		default:
 			return nil, errors.Errorf("unhandled reclaim type %q", recType)
 		}
+	// label not supported yet
+	/*case "label":
+	newReq.SetPropertyNumber(drpc.PoolPropertyLabel)
+	newReq.SetValueString(req.GetStrval())*/
```
[the remark this answers is not in the document]
Comment (mjmac) [reply src/control/server/mgmt_pool.go:297]: Hoping to add it soon, though. :) I added this and then realized that it wasn't useful without guaranteeing that the labels are unique within a system. It's something Johann has asked for as soon as possible though so I'll be adding it if I ever manage to land the new MS.

**Comment by mjmac** [comment]
Passed unit/functional tests. No point in wasting hardware time on this one.

**Pull request merged**